app/models/product.rb

```diff
@@ -34,4 +34,13 @@ class Product < ApplicationRecord
   def completed?(user)
     checks.where(user: user).present?
   end
+
+  def learning_complete
+    learning = Learning.find_by(user_id: self.user.id, practice_id: self.practice.id)
+    if learning
+      learning.change_status("complete")
```

Comment:
```suggestion
      learning.complete!
```
enumのbangメソッドを使うとシンプルにかけそうですね。`Learning#change_status` も不要になると思います。
refs: https://api.rubyonrails.org/v5.2.3/classes/ActiveRecord/Enum.html

Reply:
(Approve貰っているようなので、このPRでやらなくてもよいです!)

Reply:
ご指摘ありがとうございます。
enumいろいろな便利な書き方があるんですね!!

bangメソッドの書き方に修正しました!!